Statistics comparison

Moliholy opened the pull request:
Updated the python scripts to:

* Enable statistics comparisons between two branches
* Improve readability
* Fix bugs
* Allow users to generate csv files with their benchmark results

jblomer commented on python/statistics_comparison.py:
```diff
@@ -0,0 +1,169 @@
+    return argparser.parse_args()
+
+
+def get_benchmark_list(benchmark_string):
```
Would it be possible to make this in a way that doesn't require knowledge about the list of benchmarks?  Essentially I would provide `benchmarks/XYZ-mybenchmark` which you wouldn't need to further parse or modify,

Moliholy replied:
Of course, it would actually be simpler. I'll modify it

Moliholy commented:
Updated.
After some more clean-up steps I have successfully tested the functionality and fixed some errors in the docker container (this time not creating new ones in the process). Now it works for all four benchmarks we have. A sample generated file is pasted here:

![image](https://cloud.githubusercontent.com/assets/2722756/8106768/85c5fa74-1044-11e5-84c1-f0a0a4ed99d3.png)
